commands/proposal_submission.go

```diff
@@ -697,6 +697,14 @@ func checkNewTransferConfiguration(changes *vegapb.NewTransferConfiguration) Err
 			return errs.FinalAddForProperty("new_transfer.changes.recurring.end_epoch", ErrIsNotValid)
 		}
 
+		if f, ok := big.NewFloat(0).SetString(recurring.Factor); !ok {
+			errs.AddForProperty("new_transfer.changes.recurring.factor", ErrNotAValidFloat)
+		} else {
+			if f.Cmp(big.NewFloat(0)) <= 0 {
```

Comment:
Any particular reason why we're not using `num.DecimalFromString(recurring.Factor)`  and subsequently `f.IsZero()`?

Reply:
just consistency with the way we check this elsewhere.